packages/core/drizzle/0026_latitude_rocks.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE IF NOT EXISTS "latitude"."document_logs" (
+	"id" bigserial PRIMARY KEY NOT NULL,
+	"document_uuid" uuid NOT NULL,
+	"commit_id" bigint NOT NULL,
+	"resolved_content" text NOT NULL,
+	"parameters" json NOT NULL,
+	"custom_identifier" text,
```

Comment:
Just identifier if it's a plain text anyway

Reply:
I don't want to have two columns with the same meaning ("id" and "identifier"). This is a _custom_ field for the user to use any value as their own _identifier_.

Another field name for this could be `tag` or `custom_tag` or something like that. What do you think?